.codex/skills/.system/skill-installer/scripts/list-skills.py

```
@@ -0,0 +1,107 @@
+    try:
+        payload = _request(api_url)
+    except urllib.error.HTTPError as exc:
+        if exc.code == 404:
+            raise ListError(
+                "Skills path not found: "
+                f"https://github.com/{repo}/tree/{ref}/{path}"
+            ) from exc
+        raise ListError(f"Failed to fetch skills: HTTP {exc.code}") from exc
+    data = json.loads(payload.decode("utf-8"))
+    if not isinstance(data, list):
+        raise ListError("Unexpected skills listing response.")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)